exporter/trace/zipkin/zipkin.go

```diff
@@ -0,0 +1,194 @@
+// Copyright 2017, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package zipkin contains an exporter for Zipkin.
+//
+// Example:
+//
+// 	import (
+// 		zk "github.com/openzipkin/zipkin-go"
+// 		"go.opencensus.io/trace/adaptor/zipkin"
+// 	)
+//	...
+//		localEndpoint, err := zk.NewEndpoint("my server", myHostPort)
+// 		if err != nil {
+// 			log.Print(err)
+// 		}
+// 		exporter, err := zipkin.NewExporter("http://localhost:9411/api/v2/spans", localEndpoint)
```

Comment:
Should we should ask for the full path or the host:port?

Do we plan to support multiple API versions? @adriancole what are your thoughts?

Reply:
the endpoint url is something some mess with, for example, proxy-mounting it, etc. I think default encoding format should be v2, and when needed a separate option (v3 isn't likely for a year or two.. not even planned)

Reply:
Currently `zipkin-go` supports HTTP V2 only. However I expect to shortly have a Kafka reporter as well, which will be the preferred exporter for high throughput. Other potential stream transports for the (near) future include rabbit. The `zipkin.NewExporter` function only handles the HTTP case.

Are we going to have additional methods to handle these (and maybe rename this function to `NewHTTPExporter` or would it make sense to adjust the parameter list for the `NewExporter` function to handle alternative transports?

Reply:
In that case I'll change the exporter to take a reporter parameter, then it should be compatible with new reporter types automatically.